Samples/MobileBuyIntegration/MobileBuyIntegration/CartViewController.swift

```diff
@@ -45,14 +45,10 @@ class CartViewController: UIViewController, UITableViewDelegate, UITableViewData
 	override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
 		super.init(nibName: nibNameOrNil, bundle: nibBundleOrNil)
 
-		title = "Cart"
-
 		navigationItem.rightBarButtonItem = UIBarButtonItem(
 			title: "Reset", style: .plain, target: self, action: #selector(resetCart)
 		)
 
-		tabBarItem.image = UIImage(systemName: "cart")
```

Comment:
I've moved these to the `SceneDelegate.swift` file so that all of the tabs are configured in the same place